qt/Session.h

```diff
@@ -120,6 +122,7 @@ class Session : public QObject
 public slots:
     void addTorrent(AddData const& addme);
     void launchWebInterface();
+    void onDuplicatesTimer();
```

Comment:
Looks like a private (not public) slot to me.

Reply:
ad4c829